.github/workflows/build.yml

```diff
@@ -73,7 +79,29 @@ jobs:
         run: cargo build --release --locked --all-targets
 
       - name: Test
+        if: matrix.kind == 'test'
         run: cargo test --release --locked --all-targets
 
+      - name: Run Benchmarks
+        if: matrix.kind == 'bench'
+        run: python ./tools/benchmark.py target/release
+
+      - name: Post Benchmarks
+        if: matrix.kind == 'bench' && (github.ref == 'refs/heads/master' || github.ref == 'refs/pull/3008/merge')
+        env:
+          GH_PAT: ${{ secrets.GH_PAT }}
+        run: |
+          # Note gh-pages branch is cloned into //gh-pages/ by
+          # tools/benchmark.py, hence the following copy is ok:
+          cp -r website/* gh-pages/
+          cd gh-pages
+          # git remote add origin-ssh git@github.com:denoland/deno.git
+          git remote add origin2 https://$GH_PAT@github.com/denoland/deno.git
```

Comment:
username:password@
username:token@

Reply:
good idea - but didn't work
https://github.com/denoland/deno/pull/3008/checks?check_run_id=237718301
```
remote: Invalid username or password.
fatal: Authentication failed for 'https://github.com/denoland/deno.git/'
```

Reply:
$GH_PAT is an access token for this user?

Reply:
Yeah... I actually tried this locally and it works...